loader/loader.go

```diff
@@ -13,6 +14,15 @@ import (
 	"github.com/pkg/errors"
 )
 
+// LoadFromURL loads a docker app from an URL that should return a single-file app.
+func LoadFromURL(url string, ops ...func(*types.App) error) (*types.App, error) {
+	resp, err := http.Get(url)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to download "+url)
+	}
+	return LoadFromSingleFile(url, resp.Body, ops...)
```

Comment:
What if the response is anything but `200` ?

Reply:
I assumed it would err but you're right it doesn't. Fixed.